### PR TITLE
SerialiSe all the thingz

### LINKED ISSUE #48: Serialise vs Serialize
As discussed at the Haskell Boston meetup, the code contains both spellings of the word and there needs to be an issue and a pull request to fix that.

### PR DESCRIPTION
**Pull request by ibotty**
fixes #48.

### CODE REVIEW

**Comment by ibotty** [comment]
I did not touch the version, but of course, that's a PVP-major change.

File: tests/Tests/Reference/Implementation.hs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {-# OPTIONS_GHC -fno-warn-orphans #-}
 -----------------------------------------------------------------------------
 -- |
--- Module      : Data.Binary.Serialize.CBOR
+-- Module      : Data.Binary.Serialise.CBOR
 -- Copyright   : 2013 Simon Meier <iridcode@gmail.com>,
```
Comment (ibotty) [inline tests/Tests/Reference/Implementation.hs]: That looks odd, because it's the wrong module name all along...

**Comment by thoughtpolice** [comment]
Yeah, note the only real interface change here is in `GSerialise`, which actually was just added very recently. And we haven't done a Hackage release just yet - so no bump need here, I think.

**Comment by dcoutts** [comment]
https://www.youtube.com/watch?v=Mj2cXQXgyWE&t=43m06s

**Comment by thoughtpolice** [comment]
We'll have to implement the protective dragon mentioned in #48 later I guess.

**Pull request closed without merging**